src/jquery.nepaliDatePicker.js

```diff
@@ -473,6 +473,7 @@ var calenderFunctions = {};
                     var weekDay = $(this).data("weekday");
                     var dateText = calenderFunctions.bsDateFormat(datePickerPlugin.options.dateFormat, bsYear, bsMonth, bsDate, weekDay);
                     $element.val(dateText);
+                    $element.trigger('change');
```

Comment:
I don't think we need to trigger 'change' event here. On change date, it fire 'dateChange' event. So you can use dateChange eventListener and do task that you want to do on 'change' event.